packages/e2e-tests/config/setup-test-framework.js

```diff
@@ -154,6 +155,7 @@ beforeAll( async () => {
 
 	await trashExistingPosts();
 	await setupBrowser();
+	await activatePlugin( 'gutenberg-test-plugin-disables-the-css-animations' );
```

Comment:
Should it be placed inside `mu-plugins` instead? We have there one plugin already:

https://github.com/WordPress/gutenberg/blob/master/packages/e2e-tests/mu-plugins/disable-login-autofocus.php

Reply:
Although, it might be a bad idea as it would change the behavior when using Docker outside of e2e context.

Reply:
That was exactly my thinking :)

Reply:
Unless we disable this line in docker config:
https://github.com/WordPress/gutenberg/blob/master/docker-compose.yml#L16

Sorry about all notifications :)

Reply:
I think the e2e tests use this docker config though

Reply:
It looks like the e2e tests use a different container but when you run them locally, we often run them on the default container to test. (that's what I do), and they'll fail because animations are enabled there.